packages/compass/package.json

```diff
@@ -181,6 +181,7 @@
     "email": "compass@mongodb.com"
   },
   "dependencies": {
+    "@mongodb-js/compass-electron-menu": "^0.1.0",
```

Comment:
This gets bundled, so it's a dev dependency (SBOM collection is handled by webpack plugin), prod here is stuff that actually needs to be packaged with the application (non-prod deps will be cleaned-up from the electron app resources)